spec/requests/v1/api_conditional_request_spec.rb

```diff
@@ -132,18 +135,16 @@
 
     context "index actions" do
       let!(:project) { create_list(:project_with_contents, 2, owner: user).first }
-      let(:url) { "/api/projects" }
+      let!(:url) { "/api/projects" }
```

Comment:
Me being a bit overzealous with my let blocks when trying to figure out the spec ordering and the overriden `precondition_fails?` method.